GillianCore/debugging/adapter/inspect.ml

```diff
@@ -9,13 +9,13 @@ module DL = Debugger_log
 module Make (Debugger : Debugger.S) = struct
   open Custom_commands (Debugger)
 
-  let run dbg rpc =
-    DL.set_rpc_command_handler rpc ~name:"Threads"
+  let run ~dump_dbg dbg rpc =
```

Comment:
If `dump_dbg` is a global thing set once-and-for-all at startup, I'd make it a global variable. It's kinda polluting all signatures right now

Reply:
(Though not sure what its role is, so might no be possible)